README.rst

```diff
@@ -185,6 +189,7 @@ This will give you a mostly-headless Open edX platform. Most frontends have
 been migrated to "Micro-Frontends (MFEs)" which need to be installed and run
 separately. At a bare minimum, you will need to run the `Authentication MFE`_,
```

Comment:
This sentence says you need the Auth MFE, so should it not be listed in the table below? Same question about Learner Dashboard which is also mentioned here.

Reply:
Dashboard is needed but it turns out that AuthN is not on by default yet. I've updated the quick paragraph to be more correct and added the learner dashboard to the table below.